combirepo/repository_manager.py

```diff
@@ -155,3 +156,30 @@ def prepare(self, url, authenticator, packages_list = None):
             return repository["path"]
 
         raise Exception("Impossible happened.")
+
+    def remove_duplicates(self, repository_path):
+        rpm_list = []
+        for root, dirs, files in os.walk(repository_path):
+            for file in files:
+                if file.endswith(".rpm"):
+                    rpm_list.append(file)
+        for pkg1 in rpm_list:
```

Comment:
Let's use something more efficient that n², e.g. sort list out and then compare consequent lines